app/models/service_order.rb

```diff
@@ -17,6 +17,8 @@ class ServiceOrder < ApplicationRecord
   before_create :assign_user
   after_create  :create_order_name
 
+  virtual_attribute :v2v?, :type => :boolean
```

Comment:
instead of bringing all of these back and filtering them, could we use a scope from the API instead?

Not thrilled about the sheer number of extra records necessary to display this field.
Mentally I'm going through using sql `EXISTS ()` clauses and the like, and hoping someone will be able to find a way to do this more efficiently

```suggestion
  virtual_attribute :v2v?, :type => :boolean, :uses => :miq_requests
```

Reply:
Added, thanks :)